.travis.yml

```diff
@@ -1,7 +1,18 @@
 sudo: required
+language: node_js
+node_js:
+  - 6
+
+cache:
+  directories:
+    - ~/.npm
+    - node_modules
+install:
+  - npm install
 
 services:
   - docker
 
 script:
-  - ./run test
```

Comment:
I removed this line, because the API calls are hardcoded, and I want to use staging to run cypress (for now)